### PR TITLE
[BULK] - DocuTune remediation - Sensitive terms with GUIDs

### PR DESCRIPTION
**Pull request by meganbradley**
Applying sensitive terms with GUID changes as part of Content SFI and outlined in [Overview - Writing content securely - Platform Manual](https://review.learn.microsoft.com/en-us/help/platform/security-reference?branch=main). Changes are part of the Microsoft-wide SFI effort. Point of contact: @CelesteG

DocuTune v1.5.2.0
CorrelationId: [ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db](https://github.com/dotnet/docs/pulls?q=is%3Apr+ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db+)

#docutune

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/tools/dotnet-new-sdk-templates.md](https://github.com/dotnet/docs/blob/6001d25834f0bf9036ba99cc7fabfedfd9b95b89/docs/core/tools/dotnet-new-sdk-templates.md) | [docs/core/tools/dotnet-new-sdk-templates](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates?branch=pr-en-us-43126) |

### CODE REVIEW

**Comment by meganbradley** [comment]
#docutune-review

File: docs/core/tools/dotnet-new-sdk-templates.md [hunk 1]
```diff
@@ -251,7 +251,7 @@ The ability to create a project for an earlier TFM depends on having that versio
 
 - **`--tenant-id <ID>`**
 
-  The TenantId ID of the directory to connect to. Use with `SingleOrg` authentication. The default value is `22222222-2222-2222-2222-222222222222`.
+  The TenantId ID of the directory to connect to. Use with `SingleOrg` authentication. The default value is `aaaabbbb-0000-cccc-1111-dddd2222eeee`.
```
Comment (gewarren) [inline docs/core/tools/dotnet-new-sdk-templates.md:254]: This looks like it changes the meaning and is incorrect.

Comment (cilwerner) [inline docs/core/tools/dotnet-new-sdk-templates.md:254]:
```suggestion
  The TenantId ID of the directory to connect to. Use with `SingleOrg` authentication. The default value is `22222222-2222-2222-2222-222222222222`.
```

File: docs/core/tools/dotnet-new-sdk-templates.md [hunk 2]
```diff
@@ -418,7 +418,7 @@ The ability to create a project for an earlier TFM depends on having that versio
...
```
Comment (cilwerner) [inline docs/core/tools/dotnet-new-sdk-templates.md:421]:
```suggestion
  The TenantId ID of the directory to connect to. Use with `SingleOrg` authentication. The default value is `22222222-2222-2222-2222-222222222222`.
```

File: docs/core/tools/dotnet-new-sdk-templates.md [hunk 3]
```diff
@@ -555,7 +555,7 @@ The ability to create a project for an earlier TFM depends on having that versio
...
```
Comment (cilwerner) [inline docs/core/tools/dotnet-new-sdk-templates.md:558]:
```suggestion
  The TenantId ID of the directory to connect to. Use with `SingleOrg` authentication. The default value is `22222222-2222-2222-2222-222222222222`.
```

File: docs/core/tools/dotnet-new-sdk-templates.md [hunk 4]
```diff
@@ -764,7 +764,7 @@ Creates a web API project with AOT publish enabled. For more information, see [N
...
```
Comment (cilwerner) [inline docs/core/tools/dotnet-new-sdk-templates.md:767]:
```suggestion
  The TenantId ID of the directory to connect to. Use with `SingleOrg` authentication. The default value is `22222222-2222-2222-2222-222222222222`.
```

**Review by cilwerner — CHANGES_REQUESTED** [review]

**Comment by CelesteDG** [comment]
@gewarren - Looks like Chris reverted the changes. Should we just go ahead and close this PR? I've added the GUID 22222222-2222-2222-2222-222222222222 to the allow list since it wasn't nominated and discovered until now.

**Comment by cilwerner** [comment]
> @gewarren - Looks like Chris reverted the changes. Should we just go ahead and close this PR? I've added the GUID 22222222-2222-2222-2222-222222222222 to the allow list since it wasn't nominated and discovered until now.

Those changes still need to be committed and merged.

**Comment by CelesteDG** [comment]
PR needs to be closed. I've added the GUID to the allow list.

#please-close

CC @gewarren @meganbradley 
FYI @cilwerner @OwenRichards1

**Pull request closed without merging**